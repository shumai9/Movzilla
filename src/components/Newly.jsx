import React, {Component} from 'react';

class Newly extends Component {
  render() {
    return(null);
  }
}

export default Newly;