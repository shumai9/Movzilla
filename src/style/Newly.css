.newly {
    position: relative;
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;    
}
.newly a {
    flex: auto;
    align-items: space-around; 
    
}
.newly form, .newly p{
    z-index: 100;
    position: absolute;
    right: 0;
    top: 4rem;
    display: flex;
}
.newly p {
    left: 0;
    z-index: 100;
}