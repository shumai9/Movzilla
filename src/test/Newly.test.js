import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Newly from '../components/Newly';

it('renders with out failing', () =>{
  const div = document.createElement('div');
  ReactDOM.render(<Newly/>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', () =>{
  const tree = renderer
    .create(<Newly />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});