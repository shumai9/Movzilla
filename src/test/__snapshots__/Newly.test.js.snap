// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="new-releases"
>
  <h1>
    New Releases
  </h1>
  <div>
    <form
      onSubmit={[Function]}
    >
      <input
        className="search title"
        onChange={[Function]}
        placeholder="type movie title here"
        type="text"
      />
      <ul
        className="suggestions"
      />
    </form>
    <p>
      Total search results: 
    </p>
  </div>
  <div
    className="newly"
  />
</div>
`;
